tests/unit/routes/events/view/tickets/add-order-test.js

```diff
@@ -0,0 +1,11 @@
+import { module, test } from 'qunit';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Definition for rule 'ember-suave/no-direct-property-access' was not found.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4845812)